jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -141,6 +141,7 @@ private static void completeInitialization() {
 		/*[IF Sidecar19-SE]*/
 		System.initGPUAssist();
 
+		/*[IF JAVA_SPEC_VERSION < 19]*/
```

Comment:
you'll need to find another way to do this without deprecated APIs. All memleaks tests will fail without this. We can disable those for 19+ since its a known issue

Reply:
Now that the `Thread.getAllThreads()` API have been implemented, I think we can just use that to go through the thread list

Reply:
I've added change locally to test https://github.com/fengxue-IS/openj9/commit/9d90246af7801f5ae3da740b1612abadd399b772

Reply:
```bash
"/root/openj9-openjdk-jdk/build/linux-x86_64-server-release/images/jdk/bin/java" -version
openjdk version "19-internal" 2022-09-20
OpenJDK Runtime Environment (build 19-internal-adhoc.root.openj9-openjdk-jdk)
Eclipse OpenJ9 VM (build temp-9d90246af, JRE 19 Linux amd64-64-Bit Compressed References 20220519_000000 (JIT enabled, AOT enabled)
OpenJ9   - 5e5d8565c
OMR      - e542d0835
JCL      - 9c4cd077998 based on jdk-19+24)
make[2]: Entering directory '/root/openj9-openjdk-jdk/openj9/test'
make[3]: Entering directory '/root/openj9-openjdk-jdk/openj9/test/functional'
make[4]: Entering directory '/root/openj9-openjdk-jdk/openj9/test/functional/Java8andUp'

===============================================
Running test memleaksTest_0 ...
===============================================
memleaksTest_0 Start Time: Tue May 31 13:40:24 2022 Epoch Time (ms): 1654029624531
"/root/openj9-openjdk-jdk/build/linux-x86_64-server-release/images/jdk/bin/java" -Xshareclasses:destroyAll; "/root/openj9-openjdk-jdk/build/linux-x86_64-server-release/images/jdk/bin/java" -Xshareclasses:groupAccess,destroyAll; echo "cache cleanup done";

Attempting to destroy all caches in cacheDir /root/javasharedresources/

JVMSHRC806I Compressed references persistent shared cache "sharedcc_root" has been destroyed. Use option -Xnocompressedrefs if you want to destroy a non-compressed references cache.
JVMSHRC005I No shared class caches available
cache cleanup done
variation: NoOptions
JVM_OPTIONS:  
{ itercnt=1; \
mkdir -p "/root/openj9-openjdk-jdk/openj9/test/TKG/../TKG/output_16540296233181/memleaksTest_0"; \
cd "/root/openj9-openjdk-jdk/openj9/test/TKG/../TKG/output_16540296233181/memleaksTest_0"; \
"/root/openj9-openjdk-jdk/build/linux-x86_64-server-release/images/jdk/bin/java"   \
-Xint -verbose:shutdown -Dibm.java9.forceCommonCleanerShutdown=true -Xcheck:memory:ignoreUnfreedCallsite=zip/:dbgwrapper:unknown:wrapper -Dcom.ibm.tools.attach.logging=yes \
-cp "/root/openj9-openjdk-jdk/openj9/test/TKG/../../jvmtest/TestConfig/resources:/root/openj9-openjdk-jdk/openj9/test/TKG/../TKG/lib/testng.jar:/root/openj9-openjdk-jdk/openj9/test/TKG/../TKG/lib/jcommander.jar:/root/openj9-openjdk-jdk/openj9/test/TKG/../../jvmtest/functional/Java8andUp/GeneralTest.jar" \
org.testng.TestNG -d "/root/openj9-openjdk-jdk/openj9/test/TKG/../TKG/output_16540296233181/memleaksTest_0" "/root/openj9-openjdk-jdk/openj9/test/TKG/../../jvmtest/functional/Java8andUp/testng.xml" \
-testnames memleaksTest \
-groups level.sanity \
-excludegroups d.*.linux_x86-64,d.*.arch.x86,d.*.os.linux,d.*.bits.64,d.*.generic-all; \
if [ $? -eq 0 ] ; then echo ""; echo "memleaksTest_0""_PASSED"; echo ""; cd /root/openj9-openjdk-jdk/openj9/test/TKG/..;  else echo ""; echo "memleaksTest_0""_FAILED"; echo ""; fi; } 2>&1 | tee -a "/root/openj9-openjdk-jdk/openj9/test/TKG/../TKG/output_16540296233181/TestTargetResult";
[IncludeExcludeTestAnnotationTransformer] [INFO] EXCLUDE_FILE environment variable: /root/openj9-openjdk-jdk/openj9/test/TKG/../TestConfig/resources/excludes/latest_exclude_19.txt,/root/openj9-openjdk-jdk/openj9/test/TKG/../TestConfig/resources/excludes/feature_ojdkmh_exclude.txt
[IncludeExcludeTestAnnotationTransformer] [INFO] Processing exclude file: /root/openj9-openjdk-jdk/openj9/test/TKG/../TestConfig/resources/excludes/latest_exclude_19.txt
[IncludeExcludeTestAnnotationTransformer] [INFO] Processing exclude file: /root/openj9-openjdk-jdk/openj9/test/TKG/../TestConfig/resources/excludes/feature_ojdkmh_exclude.txt
...
... TestNG 6.14.2 by Cédric Beust (cedric@beust.com)
...

[TestHanoi] [INFO] Beginning puzzle.  Solving for 9 disks.
[TestHanoi] [INFO] Puzzle solved!
PASSED: testHanoi(9)

===============================================
    memleaksTest
    Tests run: 1, Failures: 0, Skips: 0
===============================================


===============================================
GeneralTest suite
Total tests run: 1, Failures: 0, Skips: 0
===============================================

Memory checker statistics:
Total blocks allocated = 22163 ( = most recent allocationNumber)
Total blocks freed = 16759
Total bytes allocated = 157120809
Total bytes freed = 124965508
Total unknown blocks ignored = 0
High water blocks allocated = 5549
High water bytes allocated = 33748550
Largest block ever allocated = size 2707512, allocation number 1213
Failed allocation attempts = 0
5404 allocated blocks totaling 32155301 bytes were not freed before shutdown!
The VM terminated due to exit() so unfreed blocks are expected.

memleaksTest_0_PASSED
```